config/application.php

```diff
@@ -29,13 +29,15 @@
  * Use Dotenv to set required environment variables and load .env file in root
  * .env.local will override .env if it exists
  */
-$env_files = file_exists($root_dir . '/.env.local')
-    ? ['.env', '.env.local']
-    : ['.env'];
-
-$dotenv = Dotenv\Dotenv::createUnsafeImmutable($root_dir, $env_files, false);
 if (file_exists($root_dir . '/.env')) {
+    $env_files = file_exists($root_dir . '/.env.local')
+        ? ['.env', '.env.local']
+        : ['.env'];
+
+    $dotenv = Dotenv\Dotenv::createUnsafeImmutable($root_dir, $env_files, false);
+
     $dotenv->load();
```

Comment:
@swalkinshaw, I think you're wrong. The exception could be thrown here when `$dotenv->load()` is calling. But it is already inside `if` statement. BTW, anyway I think that `.env` file should not be required.

Reply:
Ah yes, that's correct 👍